scala/spark-streaming-wordcount/README.md

```diff
@@ -0,0 +1,21 @@
+# Running Spark in Streaming Mode with Cloud Bigtable
+
+This example uses Spark streaming to pull new files from a GCS directory every 30 seconds and perform 
+a simple Spark job that counts the number of times a word appears in each new file. The Spark job uses
+Cloud Bigtable to store the results.
+
+Spark Streaming: http://spark.apache.org/docs/latest/streaming-programming-guide.html
+
+Please note that we encourage users to develop programs on their local machine, not on the VMs. User 
```

Comment:
Remove the "please note" sentence.  I think I gave that to you as advice on how to write, not to put it in the readme -- please remove from all readme's.